project/AkkaBuild.scala

```diff
@@ -79,12 +79,9 @@ object AkkaBuild extends Build {
         archivesPathFinder.get.map(file => (file -> ("akka/" + file.getName)))
       },
       Maven.projects in Maven.mvn := Seq(file("akka-samples") / "akka-docs-java-lambda"),
-      // FIXME when jenkins issue is solved, re-enable pr validation of:
-      //       Maven.mvnExec
-      //       test in Test in httpJava8Tests
       validatePullRequest <<= Seq(test in Test in stream, SphinxSupport.generate in Sphinx in docsDev,
-      test in Test in streamTestkit, test in Test in streamTests, test in Test in streamTck,
-      test in Test in httpCore, test in Test in http, test in Test in httpJavaTests,
+      Maven.mvnExec, test in Test in streamTestkit, test in Test in streamTests, test in Test in streamTck,
```

Comment:
`akka-docs-java-lambda` (previously `akka-sample-java8`) was [originaly added](https://github.com/akka/akka/pull/1879) with `.sbt` and `.pom` files. And that PR mentions that sbt does not support compiling with the different version of javac than sbt is running itself. But @jrudolph seems to have worked around that when he added `akka-http-java8-tests` or the issue was fixed in sbt.

It seems that there is no reason to keep `.pom` and extra-build-steps for java8 tests anymore. We still need maven for osgi sample though.

Builds will have to be merged anyway when we move streams to 2.3.x and finally to master. So every build change in this branch will have to be manually moved to other branches, because there will not be clean merge obviously. I think I'll create an issue with a pointers what not to forget when we move streams to 2.3.x and finally to master.